webpack/move_to_pf/LoadingState/LoadingState.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Spinner } from 'patternfly-react';
+import './LoadingState.scss';
+
+const LoadingState = ({
+  loading,
+  loadingText,
+  children,
+}) => {
+  if (loading) {
+    return (
+      <div className="loading-state">
+        <Spinner loading={loading} size="lg" />
+        <p>{loadingText}</p>
+      </div>
+    );
+  }
+
+  return children || null;
```

Comment:
If the children's default is `null` do we need this `||`?

Reply:
Good point, will remove.